.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1 @@
+/sig multicluster
```

Comment:
You can add an HTML comment like `<!-- blah blah -->` on the line above this to explain what this is and tell users not to delete it. The comment wont appear in the issue/PR body.  The k/k templates use this: https://raw.githubusercontent.com/kubernetes/kubernetes/master/.github/ISSUE_TEMPLATE.md

Reply:
Nice! Thanks for that tip.